walnuts_cpp/CMakeLists.txt

```diff
@@ -67,6 +68,26 @@ if (WALNUTS_BUILD_EXAMPLES)
   target_compile_options(test_nuts PRIVATE -O3 -Wall)
 endif()
 
+option(WALNUTS_BUILD_STAN "Build Stan example with BridgeStan" OFF)
```

Comment:
I prefer the options are all at the top so it's easier to see for someone reading the cmake file